WORKSPACE

```diff
@@ -952,7 +952,7 @@ java_import(
     jars = glob(["lib/**/*.jar"]),
 )
 """,
-    sha256 = "81e590a6854eaaf724356c0b7068502174de6457af6ec2a24624b0698eeb7362",
-    strip_prefix = "canton-community-0.21.0",
-    urls = ["https://www.canton.io/releases/canton-community-0.21.0.tar.gz"],
+    sha256 = "f9930166a52d436cf422577d68df0097b38e4b2d3b7a90e228a5117eb8494425",
```

Comment:
let’s leave it out of this PR but this should really be in `deps.bzl` not `WORKSPACE`

Reply:
I have taken a stab at moving this to `deps.bzl`

Reply:
Looks like this needs a bit more - leaving this as is for now.